Content.Server/Shuttles/ShuttleConsoleSystem.cs

```diff
@@ -72,7 +73,16 @@ private void HandlePowerChange(EntityUid uid, ShuttleConsoleComponent component,
         /// </summary>
         private void HandlePilotMove(EntityUid uid, PilotComponent component, ref MoveEvent args)
         {
-            if (component.Console == null) return;
+            if (component.Console == null)
+            {
+                DebugTools.Assert(component.Position == null);
+                EntityManager.RemoveComponent<PilotComponent>(uid);
+                return;
+            }
+
+            if (args.NewPosition.TryDistance(EntityManager, component.Position!.Value, out var distance) &&
```

Comment:
mfw the nullable is still not handled sanely
![image](https://user-images.githubusercontent.com/16618384/141124613-7918ee43-5f60-4b19-a6d7-69bbcfa5d3d1.png)

Reply:
Listen alright, if it's ever null at that stage then the server deserved what it got